### DISCUSSION TITLE
remark configuration

### DISCUSSION THREAD

**Discussion by Denice00** [remarkjs/remark#734] [Q&A] [answered]
https://github.com/remarkjs/remark-lint/blob/main/doc/rules.md

The page above under configuration shows us how to set the severity I want all of my tules to trigger and error instead of a warning, But this page does not show me how to do it in a .remarkrc config file.

Could you show me how to add the severity of the rule in my config file

**My setup is as follows:**

```
{
  "settings": {
    "bullet": "*",
    "fence": "`",
    "fences": "true",
    "listItemIndent": "tab",
    "incrementListMarker": "true",
    "resourceLink": "true",
    "rule": "-"
  },
  "plugins": [
    "remark-preset-lint-markdown-style-guide",
    ["remark-lint-list-item-indent", {"tab": true}],
    ["remark-lint-fenced-code-flag", {"allowEmpty": true}],
    ["remark-lint-maximum-line-length", false],
    ["remark-lint-unordered-list-marker-style", "*"],
    ["remark-lint-list-item-spacing", true],
    ["remark-lint-heading-increment", false],
    ["remark-lint-no-heading-punctuation", false],
    ["remark-lint-ordered-list-marker-value", "ordered"],
    "remark-preset-lint-consistent",
    "remark-preset-lint-recommended",
    ["remark-lint-final-newline", true]
  ]
}
```

**Comment by wooorm** [comment, marked as answer]
With a similar pattern to what is shown in `rules.md`, namely, arrays:

```diff
   "plugins": [
     "remark-preset-lint-markdown-style-guide",
-   ["remark-lint-list-item-indent", {"tab": true}],
+   ["remark-lint-list-item-indent", [2, {"tab": true}]],
```

However, I do not suggest you do this. remark, the CLI, has a `--frail` (`-f`) flag which treats warnings as errors. It sounds like that’s what you’re looking for

**Reply by Denice00** [reply]
Haha yeah I realised that after I tested it, it’s all good now, thank you 😊
